tools/cldr-apps/src/main/java/org/unicode/cldr/web/api/LocaleList.java

```diff
@@ -87,8 +88,43 @@ public Response normalize(
         }
 
         final LocaleNormalizerResponse r = new LocaleNormalizerResponse(ln, normalized);
+        return Response.ok().entity(r).build();
+    }
 
-        // return Response.ok(what).type(MediaType.APPLICATION_JSON_TYPE).build();
+    @Path("/combine-variants")
+    @GET
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(
+            summary = "Combine regional variants and normalize a list of Locales",
+            description = "Return a combined/normalized list of locales")
+    @APIResponses(
+            value = {
+                @APIResponse(
+                        responseCode = "200",
+                        description =
+                                "Combined/normalized response; e.g., given zh fr_BE fr_CA, return fr zh",
+                        content =
+                                @Content(
+                                        mediaType = "application/json",
+                                        schema =
+                                                @Schema(
+                                                        implementation =
+                                                                LocaleNormalizerResponse.class))),
+            })
+    public Response combineRegionalVariants(
```

Comment:
should this belong on the normalize api? if it's not really connected to forums per se?

Reply:
I'm not sure what you mean -- this is the same file as the normalize api and it shares some code with it -- and you're right, it's not connected to forums

Reply:
Sorry, i misread, sounds good!